src/platform/user/profile/vap-svc/actions/transactions.js

```diff
@@ -316,13 +318,20 @@ export const validateAddress = (
       ),
     );
   } catch (error) {
+    const errorCode = error.errors?.[0]?.code;
+    const errorStatus = error.errors?.[0]?.status;
+    if (!errorCode || !errorStatus) {
+      if (error instanceof Error) {
+        Sentry.captureException(error);
```

Comment:
My only concern with this PR is not knowing what data is inside this `error` object.

Reply:
I brought this up to the rest of FE Tools, and I agree with the following thoughts from @U-DON:

> 1. A super long try block like that kinda worries me, and I’d rather split catches into more defined error boundaries if possible.
> 1. I think this may be a case of the `apiRequest` helper not returning a standard `Error` object… Maybe worth looking into whether that might be the cause behind the issue.

Reply:
TIL we have a PII sanitizer preprocessor in Sentry: https://github.com/department-of-veterans-affairs/vets-api/blob/master/lib/sentry/processor/pii_sanitizer.rb

According to @f1337:
>Yeah we have the PII filter as a guard. In theory is logging unknown errors ideal? Maybe not, but we don't know what we don't know, and the purpose of Sentry is to catch unexpected errors. Also, if it's only going to Sentry, it's relatively easy to scrub. If we were logging unknown errors to GA, well that's a NOPE.

Reply:
Awesome... the entire purpose of this change is because of a big fat "we don't know" situation.